src/server/game/Spells/SpellEffects.cpp

```diff
@@ -1372,6 +1372,8 @@ void Spell::EffectHealthLeech(SpellEffIndex effIndex)
 
 void Spell::DoCreateItem(uint32 /*i*/, uint32 itemtype)
 {
+    TC_LOG_ERROR("spells", "DoCreateItem: %u", itemtype);
```

Comment:
This is probably just some debug log, anyway please remove it from the PR to avoid merging it by accident

Reply:
Oops, that's embarrassing. Axing.